addons/mrp/models/stock_orderpoint.py

```diff
@@ -42,6 +42,19 @@ def _compute_show_bom(self):
         for orderpoint in self:
             orderpoint.show_bom = orderpoint.route_id.id in manufacture_route
 
+    def _get_min_qty_order_possible(self):
+        """
+        Calculates the minimum quantity that can be ordered according to the qty and UoM of the BoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_order_possible()
+        if self.route_id:
+            rule_action = self.route_id.rule_ids[0].action
```

Comment:
rules_actions = self.route_id.rule_ids.mapped('action')
if 'manufacture' in rules_actions: